src/main/java/com/denizenscript/clientizen/events/EntityStartsStopsRenderingScriptEvent.java

```diff
@@ -0,0 +1,58 @@
+package com.denizenscript.clientizen.events;
+
+import com.denizenscript.clientizen.objects.EntityTag;
+import com.denizenscript.denizencore.events.ScriptEvent;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import net.minecraft.entity.Entity;
+
+public class EntityStartsStopsRenderingScriptEvent extends ScriptEvent {
+
+    // <--[event]
+    // @Events
+    // <entity> starts|stops rendering
+    //
+    // @Triggers when an entity is rendered by the client's camera. This does not mean the entity will always be visible, but within the camera's viewing frustum.
+    //
+    // @Context
+    // <context.entity> returns an EntityTag of the entity being rendered.
+    // <context.rendering> returns an ElementTag(Boolean) of whether the entity is being rendered.
+    //
+    // @Warning This event may fire very rapidly.
```

Comment:
How rapidly will it fire? I assume that's just if the player looks around a lot? in which case probably useful to specify why/when will it fire rapidly